### ISSUE TITLE
Questions about fine-tuning metric

### ISSUE THREAD

**Issue by happywinder** [PKU-ICST-MIPL/Finedefics_ICLR2025#6]
Question1:[Dataset]
In the training data for the second stage of fine-tuning, is there any overlap between the open set and closed set data?
Question2:[Metric]
In the main text, the result obtained by directly fine-tuning the model is 0.03, but in the appendix, the result obtained by direct fine-tuning is 76.13. How strange.

![Image](https://github.com/user-attachments/assets/c90ad2db-0e77-41f3-a506-26f1e1978705)

![Image](https://github.com/user-attachments/assets/db385e70-5695-4958-a39e-c4c811310ede)

**Comment by hehulingxiao** [comment]
Hello, thank you for following our work. 

Question1:[Dataset]
For each image in the training data, we formulate it as two samples: one as open-set QA sample, and the other as closed-set multiple-choice sample. Therefore, there is overlap between the open set and closed set data.
Question2:[Metric]
In the main text, "Finetune" refers to fine-tuning Idefics2 solely on additional open-set FGVR data.
In the appendix, "I.T." refers to fine-tuning Idefics2 on both closed set and open-set FGVR data.

**Comment by happywinder** [comment]
How do you test accuracy? On the open set, the closed set, or both, and then take the average?

**Comment by hehulingxiao** [comment]
In this work, we only formulate the test set as multiple-choice problem (i.e., test accuracy on the closed set).

**Comment by happywinder** [comment]
With all due respect, if the results of direct fine-tuning without contrastive learning are so close to those of contrastive learning, then what is the advantage of contrastive learning?

**Comment by hehulingxiao** [comment]
Thanks for your thoughful comments. The experiments preliminarily demonstrate the effectiveness of contrastive learning, and we are investigating scaling up the pre-training data for contrastive learning and improving attribute descriptions by integrating multiple large models.

**Comment by happywinder** [comment]
Does “finedefics” in the main text refer to fine-tuning only on open sets?

![Image](https://github.com/user-attachments/assets/ab84c6ac-8a61-4676-970d-a1caf5ee91fa)

**Comment by hehulingxiao** [comment]
Thanks for your question. "Findefics" in the main text refers to contrastive pretraining and fine-tuning on both closed set and open set.

**Comment by happywinder** [comment]
What does “pretrain” refer to?

![Image](https://github.com/user-attachments/assets/3eb8ae85-0b7f-4379-b01c-ac1eb2362b3e)

What is the difference between it and “one stage” in the main text?

![Image](https://github.com/user-attachments/assets/44fd2206-a961-422e-82fb-223e6a61f75e)

**Comment by hehulingxiao** [comment]
Thanks for your question.
"Pretrain" refers to training Idefics2 only through Stage I.
"One stage" refers to finetuning Idefics2 with a combined loss of Stage I and Stage II.
